openedx/core/djangoapps/agreements/data.py

```diff
@@ -0,0 +1,23 @@
+"""
+Public data structures for this app.
+"""
+import attr
+
+
+@attr.s(frozen=True, auto_attribs=True)
+class LTIToolsReceivingPIIData:
+    """
+    Class that stores data about the list of LTI tools sharing PII
+    """
+    lii_tools_receiving_pii: {}
```

Comment:
Once we get to the modulestore stage of the project, we may want to change this a bit to give more expected structure to each value in the dictionary, but I think this is fine as it is currently.